query.go

```diff
@@ -186,6 +220,8 @@ func translatePredicate(q query.Predicate) (sqlQuery string, sqlParams []interfa
 			switch t.Value.(type) {
 			case string:
 				sqlQuery += t.Field + " LIKE ?"
+			case int:
+				sqlQuery += t.Field + " = ?"
```

Comment:
Doesn't  `IS` work? Besides the `LIKE` query doesn't seam right here for the case of `string`. An equality query is not a like query.

Reply:
IS doesn't work. I have tried. Regarding LIKE, I also didn't think it was right, but didn't modify because I wasn't clear as to why LIKE operator was used.